packages/@glimmer/integration-tests/test/partial-rehydration-test.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  PartialRehydrationDelegate,
+  test,
+  RenderTest,
+  suite,
+  content,
+  OPEN,
+  CLOSE,
+  replaceHTML,
+  qunitFixture,
+} from '..';
+import { castToSimple } from '@glimmer/util';
+
+export class PartialRehydrationTest extends RenderTest {
```

Comment:
Can we test multiple call sites in the document?

Reply:
Yes. Added a test for this.